src/components/modebar/modebar.js

```diff
@@ -172,8 +172,12 @@ proto.createIcon = function(thisIcon) {
     icon.setAttribute('width', (thisIcon.width / iconHeight) + 'em');
     icon.setAttribute('viewBox', [0, 0, thisIcon.width, iconHeight].join(' '));
 
+    var transform = name === 'toggleSpikelines' ?
```

Comment:
A little hacky here. Ideally, the icon should have been exported 50% bigger. I opted for this hacky fix because implementing some icon `transform` setting as proposed https://github.com/plotly/plotly.js/issues/1335 would be really nice.